geissonator/openbmc-events/openbmc-events

```diff
@@ -78,7 +78,10 @@ class BMC:
         return dumps
     
     def create_dump(self):
-        r = self.session.post(self.url + 'xyz/openbmc_project/dump/action/CreateDump')
+        r = self.session.post(self.url + 'xyz/openbmc_project/dump/action/CreateDump',
+                              headers={'Content-Type': 'application/json'},
+                              data='{"data":[]}',
+                              verify=False)
```

Comment:
I don't at all like the `verify=False` part. We should let the user choose.

Reply:
Yeah, that's not even needed, just a copy/paste I guess from some other one's.